### DISCUSSION TITLE
Use Airtable as a CMS

### DISCUSSION THREAD

**Discussion by johnsicili** [webstudio-is/webstudio-community#52] [Wishlist]
[Airtable](https://airtable.com/) would be an awesome platform to manage data in. For example, add a Team Members table in Airtable, and allow Webstudio to do two things:

1. Create a list of team members on a page (aka a collection list in Webflow)
2. Create unique pages for each team member

Unfortunately, this wouldn't be a viable solution for a blog as the Rich Text editor is rather bare (basic formatting, but no images).

BTW, [Duda implemented Airtable](https://resources.duda.co/dynamic-pages-guides/airtable-collections-guide) as one of the ways to add a collection to their builder.

**Comment by kof** [comment]
closing this in favor of https://github.com/webstudio-is/webstudio-community/discussions/47
